data-raw/qc-markdowns/adult-holding-redd-and-carcass-surveys/feather-river/feather-carcass-survey-qc.Rmd

```diff
@@ -0,0 +1,539 @@
+---
+title: "Feather Carcass QC "
+author: "Inigo Peng"
+date: '2022-08-18'
+output: rmarkdown::github_document
+---
+  
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, warning = FALSE, fig.width=15, fig.height=10)
+
+library(tidyverse)
+library(lubridate)
+library(googleCloudStorageR)
+library(ggplot2)
+library(scales)
+library(readxl)
+library(janitor)
+library(hms) #?as_hms()
+library(RODBC)
+library(knitr)
+library(wesanderson)
+```
+
+
+# Feather River Carcass Data
+
+## Description of Monitoring Data
+
+**Timeframe:** 2000 to 2016
+  
+**Video Season:** 
+  
+**Completeness of Record throughout timeframe:** 
+
```

Comment:
These notes are great and exactly what I was hoping we can do. Confirm that 2011 is correct - sounds right to me. I find it strange that later years are less detailed. We should double check we aren't missing anything there.